docs/architecture/modernize-desktop/example-migration-core.md

```diff
@@ -0,0 +1,272 @@
+
+3. **Fix code and build:** Build the code in .NET Core addressing API-level differences between .NET Framework and .NET Core. If needed, update third-party packages to the ones that support .NET Core. WCF clients.
+
+4. **Run and test:** There might be differences that don't show up until the run time, so don't forget to run the application and test that everything works as expected.
+
+### Preparation
+
+#### Migrate packages.config file
+
+In a .NET Framework application, all references to external packages are declared in the *packages.config* file. In .NET Core, there's no longer the need to use the *packages.config* file. Instead, use the [PackageReference](../../core/project-sdk/msbuild-props.md#packagereference) property inside the project file to specify the NuGet packages for your app.
+
+So, you need to transition from one format to another. You can perform the update manually, taking the dependencies contained in the *packages.config* file and migrate them to the project file with the PackageReference format. Besides,
+you can leave Visual Studio do the work for you by right-clicking on the *packages.config* file and select the **Migrate packages.config to PackageReference** option.
+
+#### Verify every dependency compatibility in .NET Core
+
+Once you've migrated the package references, you must check each reference for compatibility. You can explore the dependencies of each NuGet package your application is using on [nuget.org](https://www.nuget.org/). If the package has
+.NET Standard dependencies, then it's going to work on .NET Core because .NET Core 3.1 [supports](../../standard/net-standard.md#net-implementation-support) all versions of .NET Standard. The following image shows the dependencies for
+the `Castle.Windsor` package:
+
+![Screenshot of the NuGet dependencies for the Castle.Windsor package](./media/example-migration-core/nuget-dependencies.png)
+
+To check the package compatibility, you can use the tool <http://fuget.org> that offers a more detailed information about versions and dependencies.
+
+Maybe the project is referencing older package versions that don't support .NET Core but you might find newer versions that do support it. So, updating packages to newer versions is generally a good recommendation. However, you should
+consider that updating the package version can introduce some breaking changes that would force you to update your code.
+
+What happens if you don't find a compatible version? What if you just don't want to update the version of a package because of these breaking changes? Don't worry because it's possible to depend on .NET Framework packages from a .NET
+Core application. Don't forget to test it extensively because it can cause run-time errors if the external package calls an API that isn't available on .NET Core. This is great for when you're using an old package that isn't going to be updated and you can just retarget to work on the .NET Core.
```

Comment:
```suggestion
What happens if you don't find a compatible version? What if you just don't want to update the version of a package because of these breaking changes? Don't worry because it's possible to depend on .NET Framework packages from a .NET Core application. Don't forget to test it extensively because it can cause run-time errors if the external package calls an API that isn't available on .NET Core. This is great for when you're using an old package that isn't going to be updated and you can just retarget to work on the .NET Core.
```